### ISSUE TITLE
Possible to create Pixi.spine without atlas in pixi.js version 7.3 and pixi-spine version 4.0.4 ?

### ISSUE THREAD

**Issue by MayankDhankhar** [pixijs/spine#539]
Is it possible to load spine from :
1. .json (spine)
2. no .atlas
3. individual images instead of spriteSheet whose description in skins of spine .

@ivanpopelyshev 
Hello Ivan, Your inputs on the same is highly appreciated.
Thanks.

**Comment by MayankDhankhar** [comment]
@miltoncandelero please share your inputs too.

**Comment by miltoncandelero** [comment]
I believe it is possible, but I can't remember exactly how and I no longer use this version of the plugin, so I don't know 😬 
(and even when I did use it, I didn't use that feature in particular)

**Comment by MayankDhankhar** [comment]
> I believe it is possible, but I can't remember exactly how and I no longer use this version of the plugin, so I don't know 😬 (and even when I did use it, I didn't use that feature in particular)
> 
> It had to do with the metadata you add to the spine json during load, but I can't remember the exact shape of said metadata

Ok, well no problem, Thanks for your inputs.

**Comment by MayankDhankhar** [comment]
@ivanpopelyshev - Do you know how to do it or the exact syntax for the metadata with what information to be sent to achieve the same what @miltoncandelero suggested above ?

**Comment by miltoncandelero** [comment]
Have you considered reading the comments on the loader?

https://github.com/pixijs/spine/blob/master/packages%2Floader-base%2Fsrc%2FSpineLoaderAbstract.ts#L164

**Comment by MayankDhankhar** [comment]
> Have you considered reading the comments on the loader?
> 
> https://github.com/pixijs/spine/blob/master/packages%2Floader-base%2Fsrc%2FSpineLoaderAbstract.ts#L164

Yes Sir, reading that helps but what i am trying to achieve is exactly in the link below but for pixi.js version 7.3.x and 
pixi-spine version 4.0.4  : 
[](https://github.com/pixijs/spine/blob/master/examples/dynamic_texture_atlas.md)

**Comment by MayankDhankhar** [comment]
I am able to achieve this but with the cost of extra fetch call to get data for creating texture atlas dynamically and then normally loading spine with dynamic atlas passed as metadata.
@miltoncandelero - Is there a better way of doing it, without making extra fetch call for getting json data to create texture atlas ?
